--- ubuntu-keyring_2012.05.19-pr2.diff/debian/changelog	2015-07-06 11:08:14.006365458 -0400
+++ ubuntu-keyring_2012.05.19-pr2.diff/debian/changelog	2015-07-06 11:49:44.774310951 -0400
@@ -1,3 +1,9 @@
+ubuntu-keyring (2015.07.07-pr2) precise; urgency=low
+
+  * Added Clearpath Robotics PR2 Keys
+
+ -- Devon Ash <dash@clearpathrobotics.com> Mon, 07 Jul 2015 11:49:49 +0800
+
 ubuntu-keyring (2012.05.19) quantal; urgency=low
 
   * Add 4096R/C0B21F32 Ubuntu Archive Automatic Signing Key (2012)
